.drone.yml

```diff
@@ -0,0 +1,97 @@
+kind: pipeline
+type: vm
+name: testing
+
+platform:
+  os: linux
+  arch: amd64
+
+pool:
+  use: ubuntu
+
+steps:
+  - name: build
+    image: golang
+    commands:
+      - go test ./...
+
+---
+
+kind: pipeline
+type: vm
+name: linux-amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+pool:
+  use: ubuntu
+
+steps:
+  - name: build
+    image: golang
+    commands:
+      - sh scripts/build.sh
+  - name: publish
+    image: plugins/docker
+    settings:
+      repo: plugins/nunit
```

Comment:
Why are we referring to nunit plugin repo here? 
@Ompragash : Any reason we should be adding Drone yamls here? We should have build pipelines directly in Harness.

Reply:
@hemanthmantri 
I mistakenly pushed it with the wrong plugin name, but I have corrected it now. Thank you for pointing it out.

Reply:
@hemanthmantri looks like this was created with the boilr template and we don't need them 

@vanisrikanithi could you follow the process I explained to create release pipeline in Harness CI and posted the sample pipeline reference [here](https://harness.slack.com/archives/C07NXCV17RP/p1731039410164009?thread_ts=1730989890.162719&cid=C07NXCV17RP) for ref? let me know if you find any difficulties

Reply:
@Ompragash removed the drone yaml file as it is not required.

Reply:
@Ompragash Posted a sample pipeline in the above slack group and asked a query regarding docker connector. could you please have a look and provide your comments on that?